### ISSUE TITLE
HTML minification error

### ISSUE THREAD

**Issue by fredrikekre** [tlienart/Franklin.jl#569]
The following `index.html`
```html
<!DOCTYPE html>
<html>
  <head>
    <title>JuliaCon - Julia for scripting</title>
    <meta charset="utf-8">
    <link rel="stylesheet" type="text/css" href="style.css">
  </head>
  <body>
    <textarea id="source">

      Hello.

    </textarea>

    <script type="text/javascript">
        var options = {"countIncrementalSlides":false};
        var renderMath = function() {
        // renderMathInElement(document.body);
        // or if you want to use $...$ for math,
        renderMathInElement(document.body, {delimiters: [ // mind the order of delimiters(!?)
            {left: "$$", right: "$$", display: true},
            {left: "$", right: "$", display: false},
            {left: "\\[", right: "\\]", display: true},
            {left: "\\(", right: "\\)", display: false},
        ]});
        }

        var slideshow = remark.create(options, renderMath);

        </script>
  </body>
</html>
```

is minified into
```html
<!DOCTYPE html> <title>JuliaCon - Julia for scripting</title> <meta charset=utf-8 > <link rel=stylesheet  type="text/css" href=style.css > <textarea id=source >

      Hello.

    </textarea> <script> var options = {"countIncrementalSlides":false}; var renderMath = function() { // renderMathInElement(document.body); // or if you want to use $...$ for math, renderMathInElement(document.body, {delimiters: [ // mind the order of delimiters(!?) {left: "$$", right: "$$", display: true}, {left: "$", right: "$", display: false}, {left: "\\[", right: "\\]", display: true}, {left: "\\(", right: "\\)", display: false}, ]}); } var slideshow = remark.create(options, renderMath); </script>
```

Note in particular that the closing
```html
        </script>
  </body>
</html>
```
is completely removed.

**Comment by tlienart** [comment]
Thanks, that's one that you put in assets right? So related to #568? But yeah I'll investigate, thanks for the MWE!

**Comment by fredrikekre** [comment]
Yes, exactly.

**Comment by tlienart** [comment]
should be fixed with `0.9.2` (in that files in `_assets/` are not minified)
